hmpps-prisoner-search/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/attributesearch/AttributeSearchIntegrationTest.kt

```diff
@@ -426,19 +437,19 @@ class AttributeSearchIntegrationTest : AbstractSearchIntegrationTest() {
         }
       }
 
-      webTestClient.attributeSearch(request).expectPrisoners("P1")
+      webTestClient.attributeSearch(request).expectPrisoners("P1", "P2")
     }
 
     @Test
     fun `IS with AND fields from an array`() {
       val request = RequestDsl {
         query {
           stringMatcher("alerts.alertType" IS "AT1")
-          stringMatcher("alerts.alertCode" IS "AC1")
+          stringMatcher("alerts.alertCode" IS "AC2")
         }
       }
 
-      webTestClient.attributeSearch(request).expectPrisoners("P1")
+      webTestClient.attributeSearch(request).expectPrisoners("P2")
     }
```

Comment:
Adjusted the test data so that P1 has 1 alert with type of AT1 and another alert with code AC2 - while P2 now has an alert with both type=AT1 and code=AC2.